core/src/main/scala/akka/persistence/cassandra/query/EventsByTagStage.scala

```diff
@@ -0,0 +1,295 @@
+/*
+ * Copyright (C) 2016 Typesafe Inc. <http://www.typesafe.com>
```

Comment:
2017 Lightbend :)

Reply:
Doh

Reply:
This was done via the header plugin so i updated it and all the files